doc/en/components/grids/_shared/cell-selection.md

```diff
@@ -88,10 +88,41 @@ If you want to disable cell selection you can just set `CellSelection` to **none
 
 Below are the methods that you can use in order to select ranges, clear selection or get selected cells data.
 
-<!-- Angular -->
+
+<!-- Angular, WebComponents -->
+
+### Select range
+
+`selectRange` - Select a range of cells with the API. rowStart and rowEnd should use row indexes and columnStart and columnEnd could use column index or column data field value.
+
+```ts
+const range = { rowStart: 2, rowEnd: 2, columnStart: 1, columnEnd: 1 };
+this.grid.selectRange(range);
+```
+
+<!-- end: Angular, WebComponents -->
+
```

Comment:
Clear cell selection api method information is missing for both WC and Blazor: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/cell-selection#clear-cell-selection

Reply:
It does not seem to be exposed in angular elements grid. Throws error:
clearCellSelection is not a function

Reply:
Do we have that logged?